src/components/dev-dot-content/mdx-components/mdx-code-blocks/index.tsx

```diff
@@ -29,12 +67,28 @@ export const MdxCodeBlockConfig = CodeBlockConfig
 /**
  * Adds spacing specific to Dev Dot to react-code-block <CodeTabs/>.
  */
-export function MdxCodeTabs({ className, ...restProps }: CodeTabsProps) {
+export function MdxCodeTabs({ children }: { children: ReactNode }) {
+	const childCodeBlocks = Children.toArray(children)
+
 	return (
-		<CodeTabs
-			{...restProps}
-			className={classNames(className, s.codeTabsMargin)}
-		/>
+		<Tabs>
+			{childCodeBlocks.map((child) => {
+				if (!(typeof child === 'object' && 'props' in child)) return null
+				const { mdxType, ...restprops } = child.props
+				const languageClass =
+					mdxType === 'pre' || mdxType === 'CodeBlockConfig'
+						? restprops.children.props.className
+						: `language-${restprops.language}`
+				const slugFromClass = languageClass.split('-')[1]
+				const heading = getLanguageName(slugFromClass) ?? slugFromClass
+
+				return (
+					<Tab key={slugFromClass} heading={heading} group={slugFromClass}>
+						{child}
+					</Tab>
+				)
+			})}
+		</Tabs>
```

Comment:
What do you think about splitting out this logic? Something like this (I've also taken a liberty of refactoring the `languageClass` line for clarity)

```
const setTab = (child) => {
  if (!(typeof child === 'object' && 'props' in child)) return null;

  const { mdxType, ...restprops } = child.props;
  const isCodeBlock = ['pre', 'CodeBlockConfig'].includes(mdxType);
  const languageClass = isCodeBlock ? restprops.children.props.className : `language-${restprops.language}`;
  const slugFromClass = languageClass.split('-')[1];
  const heading = getLanguageName(slugFromClass) ?? slugFromClass;

  return (
    <Tab key={slugFromClass} heading={heading} group={slugFromClass}>
      {child}
    </Tab>
  );
};

return (
  <Tabs>
    {childCodeBlocks.map(setTab)}
  </Tabs>
);
```